website/docs/theme-classic.md

```diff
@@ -48,4 +48,38 @@ module.exports = {
 
 Outbound links automatically get `target="_blank" rel="noopener noreferrer"`.
 
-## Footer
\ No newline at end of file
+## Footer
+
+## Code Blocks
```

Comment:
```suggestion
## CodeBlock
```

To ensure naming is consistent with component name

Reply:
Then we should probably enclose it in backticks?

```
## `CodeBlock`
```

Reply:
Yeah probably. Not sure what's the best